### PR TITLE
fix(tests): Lower number of content-server test executors and increase instance size

### PR DESCRIPTION
**Pull request by vbudhram**
## Because

- This might reduce flaky tests

## This pull request

- Increase instance size, reduce test executors
- Updates playwright test package

## Issue that this pull request solves

Closes: https://mozilla-hub.atlassian.net/browse/FXA-6507

## Checklist

- [x] My commit is GPG signed.
- [ ] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

### CODE REVIEW

**Comment by chenba** [comment]
@dschom has an approved but open PR with a lot of changes to the CircleCI config, so you two might need to fight it out.

File: packages/functional-tests/package.json [hunk 1]
```diff
@@ -12,14 +12,14 @@
     "test-production": "playwright test --project=production"
   },
   "devDependencies": {
-    "@playwright/test": "^1.28.1",
+    "@playwright/test": "^1.29.0",
```
Comment (dschom) [inline packages/functional-tests/package.json:15]: Just curious was a playwright update required for this change?

**Review by dschom — APPROVED** [review]
LGTM!

File: packages/functional-tests/package.json (hunk 1 above)
Comment (vbudhram) [reply packages/functional-tests/package.json:15]: Nah, just wanted to add it since `@playwright` dep was at 1.29

**Pull request merged**